src/disassemblysettingspage.ui

```diff
@@ -32,14 +32,14 @@
    <property name="bottomMargin">
```

Comment:
commenting here as github still does not allow me to comment on the commit message:

you have a typo (with -> width), and you are missing some explanatory text. please write a short summary of how things are being tied together